cmd/promtail/promtail-local-pubsub-config.yaml

```diff
@@ -0,0 +1,18 @@
+server:
+  http_listen_port: 9080
+  grpc_listen_port: 0
+
+positions:
+  filename: /tmp/positions.yaml
+
+clients:
+  - url: http://localhost:3100/loki/api/v1/push
+
+scrape_configs:
+  - job_name: pubsub-test
+    pubsub:
+      projectID: "grafanalabs-dev"
+      subscription: "dev-logs-pull"
+      credentialsPath: "/Users/kaviraj/Dropbox/notes/Grafana-keys/ops-tools-1203-05e9ed53e705.json"
```

Comment:
I don't think we need this path, we could just use the SDK the credentials should be loaded from https://cloud.google.com/docs/authentication/getting-started#setting_the_environment_variable

Reply:
yea. true. was using it before. My problem it is kinda hidden from whoever setting this config and they will know only after running Promtail for example.
 I think having it on config makes it more explicit! wdyt?

Reply:
As long as the default works (`GOOGLE_APPLICATION_CREDENTIALS `) we can keep it, since means you need to make it optional, otherwise it will different from other gcloud auth we already have like for object storage (gcs).

Reply:
Hmm!. I think default won't work. this credential is special rights to read from pubsub topic. I will check how its done on object storage. that's interesting.! Thanks for the point :)